silkworm/db/kv/grpc/client/remote_cursor_test.cpp

```diff
@@ -62,20 +62,20 @@ struct RemoteCursorTest : test_util::KVTestBase {
 
     // Execute the test preconditions: start a new Tx RPC and read first incoming message (tx_id)
     Task<::remote::Pair> start_and_read_tx_id() {
-        if (!co_await tx_rpc_.start(*stub_)) {
-            const auto status = co_await tx_rpc_.finish();
+        if (!co_await tx_rpc.start(*stub)) {
+            const auto status = co_await tx_rpc.finish();
             throw boost::system::system_error{rpc::to_system_code(status.error_code())};
         }
         ::remote::Pair tx_id_pair;
-        if (!co_await tx_rpc_.read(tx_id_pair)) {
-            const auto status = co_await tx_rpc_.finish();
+        if (!co_await tx_rpc.read(tx_id_pair)) {
+            const auto status = co_await tx_rpc.finish();
             throw boost::system::system_error{rpc::to_system_code(status.error_code())};
         }
         co_return tx_id_pair;
     }
 
-    TxRpc tx_rpc_{grpc_context_};
-    RemoteCursor remote_cursor_{tx_rpc_};
+    TxRpc tx_rpc{grpc_context_};
+    RemoteCursor remote_cursor{tx_rpc};
```

Comment:
in cases where there's a trailing underscore, I suggest to convert struct to class and try to make the members private, because this was the author's intention IMO

Reply:
done